balancer-labs/frontend-v2
# feat: 'great migration'

# Description
This pr includes changes for 'great migration' of pools

## Type of change


## How should this be tested?
Check these 5 pools (mainnet). You should see warning, and shouldn't be able to join/stake liquidity.
    ```0x8e85e97ed19c0fa13b2549309965291fbbc0048b0000000000000000000003ba,
    0xe340ebfcaa544da8bb1ee9005f1a346d50ec422e000200000000000000000396,
    0x6a5ead5433a50472642cd268e584dafa5a394490000200000000000000000366,
    0x798b112420ad6391a4129ac25ef59663a44c88bb0002000000000000000003f4,
    0x0fd5663d4893ae0d579d580584806aadd2dd0b8b000200000000000000000367```

## Visual context

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:
